honeybee_radiance_command/ra_gif.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+"""ra_gif command."""
+
+from .options.ra_gif import Ra_GIFOptions
+from ._command import Command
+import honeybee_radiance_command._typing as typing
+import honeybee_radiance_command._exception as exceptions
+
+
+class Ra_GIF(Command):
+    """ra_gif command."""
+
+    __slots__ = ('_input')
+
+    def __init__(self, options=None, output=None, input=None):
+        Command.__init__(self, output=output)
+        self._input = input
+        if options:
+            self._options = options
+        else:
+            self._options = Ra_GIFOptions()
+
+    @property
+    def options(self):
+        """RaGIF options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = Ra_GIFOptions()
+
+        if not isinstance(value, Ra_GIFOptions):
+            raise ValueError('Expected RaGIFoptions not {}'.format(value))
+
+        self._options = value
+
+    @property
+    def input(self):
+        """Radiance HDR image."""
+        return self._input
+
+    @input.setter
+    def input(self, value):
+        if value[-4:] != '.hdr':
```

Comment:
I don't know if we really need this check but if we want to check then 'HDR' should also be allowed.

```suggestion
        if value[-4:].lower() != '.hdr':
```